lib/restful_resource/strict_open_struct.rb

```diff
@@ -1,5 +1,5 @@
 module RestfulResource
   class StrictOpenStruct < ::OpenStruct
-    deprecate :dig, :[], deprecator: Deprecator.build
+    deprecate :dig, :[], deprecator: Base::Deprecator
```

Comment:
I think we can completely remove the deprecation now (https://github.com/carwow/restful_resource/commit/7db015dce61b3f2edb3f3c8143551a948f1b3e17).
Since we no longer have restful resource deprecations in our codebase according to [Honeycomb](https://ui.honeycomb.io/carwow/datasets/production/result/q4tUv3Y6a69?hideMarkers).
```suggestion
    undef_method :dig, :[]
```